binr/r2pm/d/r2pipe-go

```diff
@@ -0,0 +1,21 @@
+R2PM_BEGIN
+
+R2PM_GIT "https://github.com/radare/r2pipe.go"
+R2PM_DESC "[syspkg-r2pipe] r2pipe API for Go"
+
+R2PM_INSTALL() {
+	go get github.com/radare/radare2
+	go get github.com/radare/r2pipe.go
+	cd $GOPATH || exit 1
+	mkdir src/r_bin/
+	cd src/github.com/radare/radare2/radare2-bindings/go/
+	make
+	cp * $GOPATH/src/r_bin/ 
+	echo "r2pipe-go installed"
+}
+
+R2PM_UNINSTALL() {
+	rm -rf $GOPATH/src/r_bin
+	rm -rf $GOPATH/src/github/radare/r2pipe.go
```

Comment:
Execute `go clean -i github.com/radare/r2pipe-go` first. Also, update to `github.com/radare/r2pipe-go`.